pkg/annotation/annotation.go

```diff
@@ -2,8 +2,10 @@ package annotation
 
 // annotation on HPA and VPA resource.
 const (
-	// TortoiseNameAnnotation - VPA and HPA managed by tortoise have this label.
+	// TortoiseNameAnnotation - VPA and HPA managed by tortoise have this label. // Depricated
```

Comment:
```suggestion
	// Deprecated: TortoiseNameAnnotation - VPA and HPA managed by tortoise have this label. 
```

nit: Make it follow the golang standard.
https://go.dev/wiki/Deprecated